bindings/rust/src/bindings/mod.rs

```diff
@@ -95,23 +95,23 @@ pub fn hex_to_bytes(hex_str: &str) -> Result<Vec<u8>, Error> {
 
 /// Holds the parameters of a kzg trusted setup ceremony.
 impl KZGSettings {
-    /// Initializes a trusted setup from `FIELD_ELEMENTS_PER_BLOB` g1 points in
-    /// Lagrange form, 65 g2 points in monomial form, and
-    /// `FIELD_ELEMENTS_PER_BLOB` g1 points in monomial form.
+    /// Initializes a trusted setup from a flat array of `FIELD_ELEMENTS_PER_BLOB` G1 points in
+    /// Lagrange form, a flat array of 65 G2 points in monomial form, and a flat array of
+    /// `FIELD_ELEMENTS_PER_BLOB` G1 points in monomial form.
```

Comment:
This comment isn't exactly correct.

It says g1 Lagrange, g2 monomial, g1 monomial.
It should be g1 monomial, g1 Lagrange, then g2 monomial.

Reply:
Fixed in 75fb1ae3c